web/src/lib/utils/asset-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { api, AddAssetsResponseDto } from '@api';
+import {
+	notificationController,
+	NotificationType
+} from '$lib/components/shared-components/notification/notification';
+
+export const addAssetsToAlbum = async (
+	albumId: string,
+	assetIds: Array<string>
+): Promise<AddAssetsResponseDto> =>
+	api.albumApi.addAssetsToAlbum(albumId, { assetIds }).then(({ data: dto }) => {
```

Comment:
Can we use `async/await` syntax instead of `then` syntax to keep the code uniform across the project?